app/assets/javascripts/application/document_filter.js

```diff
@@ -65,7 +65,7 @@ if (typeof window.GOVUK === 'undefined') { window.GOVUK = {} }
     submitFilters: function (e) {
       e.preventDefault()
       var $form = documentFilter.$form
-      var $submitButton = $form.find('input[type=submit]')
+      var $submitButton = $form.find('button[type=submit]')
```

Comment:
I'm wondering ....is this button is even needed compared to just using a [change event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) on the `select`?

Reply:
I did some more research in to this one and it looks to me like the hidden submit button is still needed.

So, there's already a change event in place which fires the submission

https://github.com/alphagov/whitehall/blob/08a9c96ad27230b1c58716d0b0f952bfa1e0a445/app/assets/javascripts/application/document_filter.js#L334-L336

But the problem is when JS is disabled, then the `change` and `submit` events won't fire - and hence the previously hidden submit button is the only way to run the search.

https://user-images.githubusercontent.com/87758239/135270031-a0f94444-14b8-479e-891a-5dde7e1a3100.mov